doc/monitoring-stack-deployment/README.md

```diff
@@ -28,7 +38,86 @@ Then create secret:
 oc create secret generic additional-scrape-configs --from-file=3scale-scrape-configs.yaml=./3scale-scrape-configs.yaml
 ```
 
-5. Deploy prometheus
+3. Deploy prometheus
+
+In `prometheus.yaml` file provided, fill the `spec.externalUrl` field with the external URL. The URL template should be:
+
+```yaml
+spec:
+  ...
+  externalUrl: https://prometheus.NAMESPACE_NAME.apps.CLUSTER_DOMAIN
+```
+
+Then deploy prometheus server:
+
+```bash
+oc apply -f prometheus.yaml
+```
+
+4. Create Prometheus route
+
+```bash
+oc expose service prometheus-operated
+```
+
+5. Deploy grafana datasource
+
+```bash
+oc apply -f datasource-v4.yaml
+```
+
+6. Deploy grafana
+
+```bash
+oc apply -f grafana-v4.yaml
+```
+
+#### OpenShift versions from 4.14 and 3scale Operator version 2.15 upwards
+
+1. Install Grafana operator v5.12.0 from the Operator Hub.
```

Comment:
As I wrote before Token creation is not related to version of Grafana. Seems it can be written more simple. Also, as I wrote in different places - no need datasource-v4.yaml, and need investigate  if single grafana.yaml can be used (as probably dashboardLabelSelector will be ignored, and it's the only difference.
But main suggestion in this comment - Not make correlation of Grafana version to Token availability in 4.16 OSD

Reply:
Token creation is related to OpenShift version (4.16 specifically). 
Any suggestions on how to improve the doc?

Reply:
I have re-phrased the title of the section to be more indicative that it is 4.16.
From `OpenShift versions from 4.14 and 3scale Operator version 2.15 upwards` to `OpenShift versions from 4.16+ and 3scale Operator version 2.15+`

Reply:
I see, you have no two separate sections for v4 and v5 with all steps. Very good.
Thank you